chains/solana/contracts/tests/ccip/ccip_router_test.go

```diff
@@ -439,6 +444,9 @@ func TestCCIPRouter(t *testing.T) {
 				config.OfframpBillingSignerPDA,
 				config.FeeQuoterProgram,
 				config.FqConfigPDA,
+				config.RMNRemoteProgram,
+				config.RMNRemoteCursesPDA,
+				config.RMNRemoteCursesPDA,
```

Comment:
Should one of these be the RMN Config PDA instead of the duplicating the Curses one? 😬

Reply:
Totally 😅